projects/elife.yaml

```diff
@@ -1623,10 +1623,10 @@ iiif:
                             path: /lax:18215/elife-18215-app2-fig1-figsupp2-v2.tif
                             expected: "article/18215 article/18215v2"
                         encoded-urls:
-                            path: /lax/18215%2Felife-18215-app2-fig1-figsupp2-v2.tif
+                            path: /lax/18215%252Felife-18215-app2-fig1-figsupp2-v2.tif
```

Comment:
Other option is to always percent-encoding `%` when generating VCL/Fastly config.

Reply:
It would look more readable and less error-prone if we can do it reliably

Reply:
These are the place where to encode it:
https://github.com/elifesciences/builder/blob/master/src/buildercore/terraform.py#L307
https://github.com/elifesciences/builder/blob/master/src/buildercore/terraform.py#L314

Reply:
For example, variables of the template too:
https://github.com/elifesciences/builder/blob/master/src/buildercore/terraform.py#L363
We'd need a `VCLExpression` object for that:
```
VCLExpression('req.url ~ "%s"').format(surrogate['url'])
```